Index: app/src/main/java/com/example/burgerapp/ui/theme/Color.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.burgerapp.ui.theme\r\n\r\nimport androidx.compose.ui.graphics.Color\r\n\r\nval Purple80 = Color(0xFFD0BCFF)\r\nval PurpleGrey80 = Color(0xFFCCC2DC)\r\nval Pink80 = Color(0xFFEFB8C8)\r\n\r\nval Purple40 = Color(0xFF6650a4)\r\nval PurpleGrey40 = Color(0xFF625b71)\r\nval Pink40 = Color(0xFF7D5260)\r\nval CherryRed = Color(0xFFB22222)\r\nval WhiteText = Color(0xFFFFFFFF)\r\nval LightGray = Color(0xFFF5F5F5)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/burgerapp/ui/theme/Color.kt b/app/src/main/java/com/example/burgerapp/ui/theme/Color.kt
--- a/app/src/main/java/com/example/burgerapp/ui/theme/Color.kt	(revision f4f4b0d82101cd298596665f082a14350fc5cc00)
+++ b/app/src/main/java/com/example/burgerapp/ui/theme/Color.kt	(date 1758087371384)
@@ -12,3 +12,7 @@
 val CherryRed = Color(0xFFB22222)
 val WhiteText = Color(0xFFFFFFFF)
 val LightGray = Color(0xFFF5F5F5)
+val LightCherryRed = Color(0xFFFF6F61)
+// ui/theme/Color.kt
+
+
Index: build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>// Top-level build file where you can add configuration options common to all sub-projects/modules.\r\nplugins {\r\n    alias(libs.plugins.android.application) apply false\r\n    alias(libs.plugins.kotlin.android) apply false\r\n    alias(libs.plugins.kotlin.compose) apply false\r\n    id(\"com.google.gms.google-services\") version \"4.4.3\" apply false}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle.kts b/build.gradle.kts
--- a/build.gradle.kts	(revision f4f4b0d82101cd298596665f082a14350fc5cc00)
+++ b/build.gradle.kts	(date 1758102600045)
@@ -3,4 +3,9 @@
     alias(libs.plugins.android.application) apply false
     alias(libs.plugins.kotlin.android) apply false
     alias(libs.plugins.kotlin.compose) apply false
-    id("com.google.gms.google-services") version "4.4.3" apply false}
+    id("com.google.gms.google-services") version "4.4.3" apply false
+
+
+}
+
+
